nucliadb_node/src/merge/scheduler.rs

```diff
@@ -54,11 +60,19 @@ pub struct MergeScheduler {
 impl MergeScheduler {
     pub fn new(shard_cache: Arc<ShardWriterCache>, settings: Settings) -> Self {
         Self {
-            work_queue: Mutex::new(WorkQueue::new()),
             shard_cache,
             settings,
+            work_queue: Mutex::new(WorkQueue::new()),
             condvar: Condvar::default(),
             shutdown: AtomicBool::new(false),
+            max_nodes_in_merge: match std::env::var(MAX_NODES_IN_MERGE) {
+                Ok(v) => v.parse().unwrap_or(50_000),
+                Err(_) => 50_000,
```

Comment:
We might want to set a higher default here, so things start happening right away. Maybe it's enough with 100k? Anyway, we can always tweak via env variable later.

Reply:
I didn't want to increase that limit right away, doing it through the env vars life seemed safer. But if you think is not worth I dont mind changing it.